pkg/pillar/types/zedroutertypes.go

```diff
@@ -791,7 +793,7 @@ func (portConfig *DevicePortConfig) IsAnyPortInPciBack(
 	for _, port := range portConfig.Ports {
 		// XXX this assumes that ioBundle.Name is the ifname known
```

Comment:
Could you please update the comment.

Reply:
Done.. Removed it. It is no longer needed.